src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -66,6 +68,7 @@ TriggeredSubscription::TriggeredSubscription
   format            = _format;
   reference         = _reference;
   attrL             = _attrL;
+  cacheSubReference = NULL;
```

Comment:
Not sure what you are asking for here ...

The pointer to the Subscription in the SubscriptionCache is used for lastNotificationTime and pendingNotifications.

I *could* have implemented this doing some kind of linear search for every triggeredSubscription, but it would be perhaps 1000 times slower ...

NTC